Assets/MRTK/Core/Providers/Hands/BaseHandVisualizer.cs

```diff
@@ -236,6 +240,7 @@ public virtual void OnHandMeshUpdated(InputEventData<HandMeshInfo> eventData)
                     }
 
                     handMeshFilter.transform.SetPositionAndRotation(eventData.InputData.position, eventData.InputData.rotation);
+                    handMeshFilter.transform.parent = transform;
```

Comment:
Do we need to set this on every `OnHandMeshUpdated` or should it just be set in the initial spawn logic? It also looks like we weren't currently enforcing this, so hopefully there's no side effects?

Reply:
Fixed!